test/JDBC/upgrade/13_4/schedule

```diff
@@ -101,6 +101,7 @@ Test-sp_helpsrvrolemember
 BABEL-404
 BABEL-493
 BABEL-621
+BABEL-736
```

Comment:
Do we really want to test previous behavior? If so, follow this (https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/270/files), otherwise please put this only in the latest schedule file.